sharding/sharding.go

```diff
@@ -167,7 +173,7 @@ func (r *ShardingFerry) Run() {
 	r.Ferry.Throttler.SetDisabled(false)
 
 	metrics.Measure("CutoverUnlock", nil, 1.0, func() {
-		err = r.config.CutoverUnlock.Post(client)
+		err = r.config.CutoverUnlock.Post(&client)
```

Comment:
Any reason the type changed?

Reply:
This was changed from
```go
client := &http.Client{}
```
to a var declaration block
```go
var (
	err          error
	client       http.Client
	cutoverStart time.Time
)
```